GDAXSharp/Network/Authentication/Authenticator.cs

```diff
@@ -17,5 +23,21 @@ public class Authenticator : IAuthenticator
         public string UnsignedSignature { get; }
 
         public string Passphrase { get; }
+
+        public string ComputeSignature(HttpMethod httpMethod, string secret, double timestamp, string requestUri, string contentBody = "")
```

Comment:
I am of the opinion that this still belongs in the `HttpRequestMessageService` because that's really the only thing that cares about it.

Reply:
I somewhat agree, however, for the authentication of the user channel this function is needed and since this function is always needed when authenticating I thought this could be an appropriate place. Also initializing `HttpRequestMessageService` just for `ComputeSignature` seems not the right way. Please let me know what you think of it :)

Reply:
I somewhat agree, however, for the authentication of the user channel this function is needed and since this function is always needed when authenticating I thought this could be an appropriate place. Also initializing `HttpRequestMessageService` just for `ComputeSignature` seems not the right way. Please let me know what you think of it :)